test/lint/lint-python.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/usr/bin/env bash
```

Comment:
Adding a specific dependency on bash is acceptable, because other linter scripts already rely on bash-isms anyway, I suppose?